src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -1,40 +1,224 @@
-import React from "react";
+import React, { useState } from "react";
 import {
   Button,
+  Dropdown,
+  DropdownItem,
+  DropdownToggle,
+  Label,
   PageSection,
   PageSectionVariants,
+  Split,
+  SplitItem,
+  Tab,
+  TabContent,
+  Tabs,
+  TabTitleText,
   Text,
   TextContent,
 } from "@patternfly/react-core";
-import { Link, useLocation } from "react-router-dom";
+import { Link, useHistory, useLocation } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { Breadcrumb } from "@app/components/Breadcrumb/Breadcrumb";
+import { CaretDownIcon } from "@patternfly/react-icons";
+import { TableWithPagination } from "@app/components/TableWithPagination/TableWithPagination";
+import { IRow, IRowData } from "@patternfly/react-table";
+import { Instance } from "../../../types/Instance";
+import { formatDistance } from "date-fns";
+import "./InstancePage.css";
 
 const InstancePage = () => {
+  const { t } = useTranslation(["openbridgeTempDictionary"]);
   const location = useLocation();
+  const history = useHistory();
+
+  const processorsTabRef = React.createRef<HTMLElement>();
+  const accessTabRef = React.createRef<HTMLElement>();
+
+  const [activeTabKey, setActiveTabKey] = useState<number | string>(0);
+  const [isDropdownActionOpen, setIsDropdownActionOpen] =
+    useState<boolean>(false);
+
+  const instanceName = `My instance`; // @TODO retrieve it from API
+
+  const handleTabClick = (
+    _: React.MouseEvent<HTMLElement, MouseEvent>,
+    eventKey: number | string
+  ) => {
+    setActiveTabKey(eventKey);
+  };
+
+  const onDetailsClick = () =>
+    // @TODO missing action to perform when clicking on details action
+    {};
+
+  const onDeleteClick = () =>
+    // @TODO missing action to perform when clicking on delete action
+    {
+      history.push(`/`);
+    };
+
+  const processorsOverviewColumns = [
+    {
+      accessor: "name",
+      label: t("common.name"),
+      formatter: (value: IRowData, row?: IRow) => {
+        const processorId = (row as Instance)?.id;
+        return (
+          <Link
+            data-testid="tableProcessors-linkProcessor"
+            to={`${location.pathname}/processor/${processorId}`}
```

Comment:
In this case, it will happen that the link will be directed to an undefined route. I think it is perfectly fine to have a wrong result, from a wrong starting situation (w/o unblocking the user flow)